tracer/src/Datadog.Trace/TracerManagerFactory.cs

```diff
@@ -268,6 +268,13 @@ protected virtual ITraceSampler GetSampler(ImmutableTracerSettings settings)
             // simply overriden with the remote configuration. Instead, remote rules are merged with local rules,
             // with remote rules taking precedence.
 
+            if (settings.AppsecStandaloneEnabledInternal)
+            {
+                var samplerStandalone = new TraceSampler(new TracerRateLimiter(1, 60_000));
```

Comment:
Have you tested that this works the way you expect? I think this will limit _all_ traces (APM and ASM traces) to 1 trace per minute. IIUC, according to the RFC you want:
- keep all ASM traces
- allow at most 1 APM trace per minute to keep the service alive

Reply:
I see this code was removed. Are we not implementing this limiter part of the RFC yet?